.gitignore

```diff
@@ -1 +1,2 @@
 .vercel
+index.html
```

Comment:
Since the `index.html` is already tracked, adding it in `.gitignore` didn't have any effect 🙂 
It probably needs to be deleted and commit first to ignore it